tyssue/core/sheet.py

```diff
@@ -273,6 +275,75 @@ def extract_bounding_box(
         subsheet.reset_topo()
         return subsheet
 
+    def get_force_inference(self, coords=None):
+        """Measure force based on Brodland method.
+
+        g_gamma_matrix*tension_vector = 0
+        Equation is homogenous and to avoid tension_vectors = 0,
+        Construction and solve the constrained least-squares equation system
+        [[g_gamma_matrix.T g_gamma_matrix, C^T_1],
+         [C_1, 0]]
+         où C1 = {1....1}
+
+        """
+        if coords is None:
+            coords = self.coords
+        ndim = self.dim
```

Comment:
maybe `ndim` should be `len(coords)` here (if we do strange projections for ex.).